Implement xdebug.discover_client_host, fix bug with WSL2 docker-inside getting correct host.docker.internal, fixes #4803 (#4852)
ddev/ddev: docs/content/users/install/ddev-installation.md

      All Windows 10/11 editions, including Windows 10 Home support WSL2.
    * WSL2 offers a faster, smoother experience.  
      It’s vastly more performant, and you’re less likely to have obscure Windows problems.

>     * Execute DDEV commands inside WSL2 (inside Ubuntu, for example). You will never execute DDEV comamnds on the Windows side, in PowerShell or Git Bash, for example.

Reviewer: Follow the format used with the rest of this list and avoid the typo:

```suggestion
    * Execute DDEV commands inside WSL2.   
    You’ll want to run DDEV commands inside Ubuntu, for example, and never on the Windows side in PowerShell or Git Bash.
```

      It’s vastly more performant, and you’re less likely to have obscure Windows problems.

    * Execute DDEV commands inside WSL2.   

>       u:0You’ll want to run DDEV commands inside Ubuntu, for example, and never on the Windows side in PowerShell or Git Bash.

Reviewer: @rfay This `u:0` doesn’t seem like it’s for humans.

Reviewer: `z:4`! 😠
